backend/pkg/api/router.go

```diff
@@ -94,8 +94,12 @@ func addRoutes(hs *handlers.HandlerService, publicRouter, internalRouter *mux.Ro
 		{http.MethodGet, "/product-summary", nil, hs.InternalGetProductSummary},
 
 		{http.MethodPost, "/users", nil, hs.InternalPostUsers},
-
+		{http.MethodPut, "/users/confirm/{token}", nil, hs.InternalPostUserConfirm},
```

Comment:
Should be a POST request.
Minor: Also I'd like to stay RESTful and avoid verbs in URLs if possible, so I'd prefer this path:
```suggestion
		{http.MethodPost, "/users/email-confirmations/{token}", nil, hs.InternalPostUserConfirm},
```